yashsachdev.HabitTracker/DisplayClass.cs

```diff
@@ -0,0 +1,242 @@
+        HabitRepo habit = new HabitRepo();
+        UserRepo userRepo = new UserRepo();
+        using (SqliteConnection cnn = new SqliteConnection(DatabaseClass.connectionString))
+        {
+            cnn.Open();
+            Console.WriteLine("Enter habit Name you want to update");
+            string habitname = UserInput.GetHabitName();
+            this.UserId = userRepo.GetIdFromEmail(this.Email);
+            if (UserInput.ValidateName(habitname) == false)
+            {
+                Console.WriteLine("Please enter correct details");
+                return;
+            }
+
+            var habitCount = habit.CountofHabit(this.Email, habitname);
+            if (habitCount == 0)
+            {
+                Console.WriteLine("Habit not found. Please Insert Habit first.");
+                return;
+            }
+            if (!habitEnrollRepo.CheckIfHabitExistsForUser(habitname, this.UserId))
+            {
+                Console.WriteLine("You do not have a habit with that name. Please try again.");
+                return;
+            }
+            Console.WriteLine($"Enter the values you want to update");
+            string resunit = UserInput.GetUnitMeasurement();
+            string habitUnit = UserInput.GetHabitUnit(resunit);
+            var habit_id = habitEnrollRepo.GetHabitId(habitname, this.UserId);
+            habitEnrollRepo.UpdateUserHabit(this.Email, habitname, habitUnit, habit_id);
+            habitEnrollRepo.DisplayUserHabit(this.Email);
+        }
+    }
+    private void DeleteHabit()
+    {
+        HabitEnrollRepo habitEnrollRepo = new HabitEnrollRepo();
+        HabitRepo habitRepo = new HabitRepo();
+        string habitname = UserInput.GetHabitName();
+        if (!habitEnrollRepo.CheckIfHabitExistsForUser(habitname, UserId))
+        {
+            Console.WriteLine("You do not have a habit with that name. Please try again.");
+            return;
+        }
+        var habit_Id = habitEnrollRepo.GetHabitId(habitname, UserId);
+        habitRepo.DeleteHabit(habit_Id);
+        Console.WriteLine($"Deleted {habitname}");
+        Console.WriteLine("Press any key to continue...");
+        Console.ReadKey();
+    }
+    private void Logout()
+    {
+        this.LoggedIn = false;
+    }
+    private void ViewHabits()
+    {
+        HabitEnrollRepo habitEnrollRepo = new HabitEnrollRepo();
+        habitEnrollRepo.DisplayUserHabit(this.Email);
+        Console.WriteLine("Press any key to continue...");
+        Console.ReadKey();
+    }
+    private void InsertHabit()
+    {
+        string habitName = UserInput.GetHabitName();
+        if (UserInput.ValidateName(habitName) == false)
+        {
+            Console.WriteLine("Please enter valid habit");
+            return;
+        }
+
+        HabitRepo habitRepo = new HabitRepo();
+        UserRepo userRepo = new UserRepo();
+        HabitEnrollRepo habitEnrollRepo = new HabitEnrollRepo();
```

Comment:
Not mandatory, but add a line between declarations and expressions for better readability.